src/modules/AuthModule/components/ForgetPass/ForgetPass.tsx

```diff
@@ -1,8 +1,163 @@
-import React from 'react'
+import {
+  Alert,
+  InputLabel,
+  OutlinedInput,
+} from "@mui/material";
+import Box from "@mui/material/Box";
+import Button from "@mui/material/Button";
+import CssBaseline from "@mui/material/CssBaseline";
+import Grid from "@mui/material/Grid";
+import Link from "@mui/material/Link";
+import Paper from "@mui/material/Paper";
+import { createTheme, ThemeProvider } from "@mui/material/styles";
+import Typography from "@mui/material/Typography";
+import { emailValidation } from "./../../../../validations/validations";
+import { SubmitHandler, useForm } from "react-hook-form";
+import { FormValuesForgetPass } from "../../../../interfaces/Auth";
+import { axiosInstance } from "../../../../axiosConfig/axiosInstance";
+import { useState } from "react";
+import { toast } from "react-toastify";
```

Comment:
We might look for the snackbar component from MUI
https://mui.com/material-ui/react-snackbar/